frontend/.eslintrc.js

```diff
@@ -11,5 +11,6 @@ module.exports = {
     quotes: ["error", "double"],
     "no-console": "error",
     "no-debugger": "error",
+    "@typescript-eslint/no-explicit-any": ["error", { ignoreRestArgs: true }],
```

Comment:
This rule only allows usages of `any` when declaring a function that takes in any number of parameters:
e.g. ```function a(...args: any[]) { ... }```